source/_patterns/01-atoms/svgicon/_svgicon.scss

```diff
@@ -0,0 +1,14 @@
+// All component Sass needs non-printing base config
+@import '../../00-protons/variables';
+@import '../../../../apps/pl/scss/_scss2json';
+@import './scss/_icons-generated';
+
+// Loop over every SVG and create a CSS class for it
```

Comment:
Let's also generate all the `.color-fill--{colorname}` classes here as well, allowing us to colorize the SVG files by class name from the design system.

Reply:
Added in c4c164c.

The twig svgfill now accepts one of the keys defined in $theme-colors as an argument which applies the related BEM class for colorizing the SVG.